src/checkqueue.h

```diff
@@ -173,10 +173,7 @@ class CCheckQueue
 
```

Comment:
in 356b2a5b946304165a8f762dc6a2d62f3e12139d: I don't really like using move semantics on a mutable reference. This makes code fragile, because the caller may not realize the memory is stolen. It would be better to adjust the signature of this function from `&` to `&&`.

Reply:
Agree.

Reply:
> It would be better to adjust the signature of this function from `&` to `&&`.

Thanks! Added a dedicated commit.